cypress/integration/detectors_spec.js

```diff
@@ -0,0 +1,250 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  PLUGIN_NAME,
+  TWENTY_SECONDS_TIMEOUT,
+  TEST_INDEX,
+  TEST_FIELD_MAPPINGS,
+} from '../support/constants';
+
+describe('Detectors', () => {
```

Comment:
Let's call this `Detector` since the tests are describing actions on single detector?